src/Symfony/Component/Intl/Resources/bin/update-data.php

```diff
@@ -178,8 +178,7 @@
 $compiler = new GenrbCompiler($genrb, $genrbEnv);
 $config = new GeneratorConfig($sourceDir.'/data', $icuVersionInDownload);
 
-// Don't wrap "/data" in realpath(), in case the directory does not exist
```

Comment:
well, there is no realpath at all anymore. And btw, this was removed because @fabpot asked for it.